src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -56,6 +56,13 @@
   (->> (select-keys config-map #{:allow :allow-ip :deny :deny-ip})
        (reduce #(add-individual-acl (first %2) (second %2) %1) rule)))
 
+(defn add-query-params
+  "Add any query parameters specified in configuration to the rule."
+  [rule {:keys [query-params]}]
+  (if query-params
```

Comment:
Ohhh I see, yes actually I think that'll work as `reduce-kv` will just return `rule` if the `coll` is empty or nil. Nice catch.